CHANGELOG.md

```diff
@@ -1,5 +1,38 @@
 # CHANGELOG
 
+## 2.5.0 (2023-07-13)
+
+MLflow 2.5.0 includes several major features and improvements
+
```

Comment:
TODO: Add a section for mlflow auth

Reply:
Ref: https://github.com/mlflow/mlflow/releases/tag/v1.27.0 where we released MLflow pipelines.

Reply:
it looks great! Well done :)